juriscraper/pacer/appellate_attachment_page.py

```diff
@@ -0,0 +1,211 @@
+from __future__ import print_function
+
+import pprint
+import re
+import sys
+
+from lxml import html
+
+from juriscraper.lib.html_utils import strip_bad_html_tags_insecure
+from juriscraper.lib.string_utils import force_unicode
+from .reports import BaseReport
+from ..lib.log_tools import make_default_logger
+
+logger = make_default_logger()
+
+
+class AppellateAttachmentPage(BaseReport):
+    """An object for querying and parsing the appellate att. page report."""
+
+    PATH = "doc1/"
```

Comment:
I thought it was `docs1` on appellate, not `doc1`? Is this tested? If not, maybe we should remove it and raise a NotImplemented exception for now.

Reply:
Correct. Although I was always prepared for both to work, although that's not the case.

Reply:
It is doc1 for the regular AttachmentPage - is this an error that should be changed as well?

Reply:
I think district uses doc1 and appellate uses docs1?

Reply:
Thanks